src/AnnotationDialog.js

```diff
@@ -614,7 +614,27 @@ class AnnotationDialog extends EventEmitter {
             hour: '2-digit',
             minute: '2-digit'
         });
-        const text = util.htmlEscape(annotation.text);
+        const annotationText = annotation.text;
+        // Respect newline characters
+        const newlineRegex = /\r\n|\n\r|\n|\r/g;
+        const newlineList = annotationText.replace(newlineRegex, '\n').split('\n');
```

Comment:
Does this turn multiple new lines into one? I was able to match 3 new lines, and then it looks like we're replacing that with one and then splitting on it to add to the array

Reply:
From testing this, it does split it into multiple newlines. I was able to place several newlines in a row and successfully render them as expected. I'll double check

Reply:
<img width="283" alt="screen shot 2017-12-14 at 10 01 49 am" src="https://user-images.githubusercontent.com/763560/34001518-d473aaec-e0b5-11e7-908d-44eeebc07bfe.png">

Reply:
I believe you but I don't understand why that works. It looks like you're targeting multiple newlines and then replacing them with 1.